modules/network/src/main/java/org/apache/ignite/network/scalecube/ScaleCubeClusterServiceFactory.java

```diff
@@ -143,10 +151,8 @@ public void onMembershipEvent(MembershipEvent event) {
 
                 cluster.startAwait();
 
-                // emit an artificial event as if the local member has joined the topology (ScaleCube doesn't do that)
-                var localMembershipEvent = MembershipEvent.createAdded(cluster.member(), null, System.currentTimeMillis());
-
-                topologyService.onMembershipEvent(localMembershipEvent);
+                // update info about the local member (ScaleCube doesn't emit events about local members)
+                topologyService.updateLocalMember(cluster.member(), null);
```

Comment:
This is not a correct change. We rely on the fact that adding a local member will trigger topology listeners (`onAppeared`), I wonder if the tests will even pass